docs/contributing/assets/allowed_actions.json

```diff
@@ -16,6 +16,22 @@
       },
       "comment": ""
     },
+    {
+      "name": "golangci/golangci-lint-action",
+      "versions": [
+        "3cfe3a4abbb849e10058ce4af15d205b6da42804",
+        "v4"
+      ],
+      "repository": "https://github.com/golangci/golangci-lint-action",
+      "marketplace": "https://github.com/marketplace/actions/run-golangci-lint",
+      "security_review_performed": true,
+      "3rd_party_tool": {
+        "tool": "golangci-lint",
+        "pinned_version": "???",
```

Comment:
According to the [documentation](https://github.com/marketplace/actions/run-golangci-lint), it is possible to use a dedicated version of the underlying tool [golangci-lint](https://github.com/golangci/golangci-lint). Please add which version you have used to test and will be used in production. It is required to use dedicated versions of 3rd party tools whenever possible.